enginetest/queries/priv_auth_queries.go

```diff
@@ -1032,6 +1034,68 @@ var ServerAuthTests = []ServerAuthenticationTest{
 			},
 		},
 	},
+	{
+		Name: "Create User with mysql_clear_password plugin specification",
+		SetUpScript: []string{
+			"CREATE USER ctpuse@localhost IDENTIFIED WITH mysql_clear_password",
+			"GRANT ALL ON *.* TO ctpuse@localhost WITH GRANT OPTION;",
+		},
+		Assertions: []ServerAuthenticationTestAssertion{
+			{
+				Username:    "ctpuse",
+				Password:    "",
+				Query:       "SELECT * FROM mysql.user;",
+				ExpectedErr: true,
+			},
+			{
+				Username:    "ctpuse",
+				Password:    "rightpassword",
+				Query:       "SELECT * FROM mysql.user;",
+				ExpectedErr: false,
+			},
+		},
+	},
+	{
+		Name: "Create User with jwt plugin specification",
+		SetUpScript: []string{
+			"CREATE USER `test-user`@localhost IDENTIFIED WITH authentication_dolt_jwt AS 'jwks=testing,sub=test-user,iss=dolthub.com,aud=some_id';",
+			"GRANT ALL ON *.* TO `test-user`@localhost WITH GRANT OPTION;",
+		},
+		SetUpFunc: func(ctx *sql.Context, t *testing.T, engine *sqle.Engine) {
+			engine.Analyzer.Catalog.MySQLDb.SetJwksConfig([]mysql_db.JwksConfig{
+				{
+					Name:   "testing",
+					Source: "testdata/test_jwks_keys.json", // TODO: Use url only?
```

Comment:
Yeah, could this be a `file://` URL maybe? Or is that not really better / implicit file if it doesn't start with `http{,s}`?